src/state/fragments/actions.js

```diff
@@ -199,6 +206,7 @@ export const sendPostFragment = (fragment, saveType) => async (dispatch) => {
       { id: 'fragment.created' },
       TOAST_SUCCESS,
     ));
+    dispatch(clearSelectedFragment());
```

Comment:
should not same `clearSelectedFragment` be dispatched also during `put` and `delete` fragment actions?

Reply:
`clearSelectedFragment` invokes once user leaves the form (goes to another page). I used willunmount cycle. Is that okay?

Reply:
yes its ok, but why do we have it here then? just wondering

Reply:
hmm good point. I'll remove that